index.js

```diff
@@ -10,6 +10,7 @@ export const Rels = {
 	departments: 'https://api.brightspace.com/rels/departments',
 	displayName: 'https://api.brightspace.com/rels/display-name',
 	enrollments: 'https://api.brightspace.com/rels/enrollments',
+	filter: 'https://api.brightspace.com/rels/filter',
```

Comment:
Once you've entered into a filter collection, you choose the `filter` you want to apply